edge-modules/load-gen/Program.cs

```diff
@@ -77,8 +77,14 @@ static async Task GenerateMessageAsync(ModuleClient client, Guid batchId)
                     messageIdCounter++;
                     message.Properties.Add("sequenceNumber", messageIdCounter.ToString());
                     message.Properties.Add("batchId", batchId.ToString());
+                    message.Properties.Add("trackingId", trackingId);
```

Comment:
What will this tracking Id be used for?

Reply:
it is used for connectivity test to uniquely identify all test results belong to this test run.  This will help to easily find all results upload to log analytics using this tracking id.